runtime-modules/proposals/discussion/src/benchmarking.rs

```diff
@@ -306,6 +308,40 @@ benchmarks! {
         assert_last_event::<T>(RawEvent::PostUpdated(post_id, caller_member_id, thread_id, new_text).into());
     }
 
+    delete_post {
+        // We do this to ignore the id 0 because the `Test` runtime
+        // returns 0 as an invalid id but 1 as a valid one
+        let (_, _) = member_account::<T>("caller_member", 0);
+        let (account_id, caller_member_id) = member_account::<T>("caller_member", 1);
+
+        // Worst case scenario there is a council
+        elect_council::<T>(2);
```

Comment:
I failed to grasp the council effect on the worst-case here. Do you mind commenting on it, please?

Reply:
This was a left over it was not needed.

Reply:
Ok. There are other such code lines in this benchmarking file. Remove them too, please, if necessary.